deltachat-ios/AppDelegate.swift

```diff
@@ -17,293 +17,305 @@ var mailboxPointer: UnsafeMutablePointer<dc_context_t>!
-      let value = kv.count > 1 ? kv[1] : ""
-      return DBCustomVariable(name: kv[0], value: value)
-    }
-
-    DBDebugToolkit.add(info)
-  }
-
-  @objc func reachabilityChanged(note: Notification) {
-    let reachability = note.object as! Reachability
-
-    switch reachability.connection {
-    case .wifi, .cellular:
-      logger.info("network: reachable", reachability.connection.description)
-      dc_maybe_network(mailboxPointer)
-
-      let nc = NotificationCenter.default
-      DispatchQueue.main.async {
-        nc.post(name: dcNotificationStateChanged,
-                object: nil,
-                userInfo: ["state": "online"])
-      }
-    case .none:
-      logger.info("network: not reachable")
-      let nc = NotificationCenter.default
-      DispatchQueue.main.async {
-        nc.post(name: dcNotificationStateChanged,
-                object: nil,
-                userInfo: ["state": "offline"])
-      }
-    }
-  }
-
-  // MARK: - BackgroundTask
-
-  func registerBackgroundTask() {
-    logger.info("background task registered")
-    backgroundTask = UIApplication.shared.beginBackgroundTask { [weak self] in
-      self?.endBackgroundTask()
-    }
-    assert(backgroundTask != .invalid)
-  }
-
-  func endBackgroundTask() {
-    logger.info("background task ended")
-    UIApplication.shared.endBackgroundTask(backgroundTask)
-    backgroundTask = .invalid
-  }
-
-  // MARK: - PushNotifications
-
-  func registerForPushNotifications() {
-    UNUserNotificationCenter.current().delegate = self
-
-    UNUserNotificationCenter.current()
-      .requestAuthorization(options: [.alert, .sound, .badge]) {
-        granted, _ in
-        logger.info("permission granted: \(granted)")
-        guard granted else { return }
-        self.getNotificationSettings()
-      }
-  }
-
-  func getNotificationSettings() {
-    UNUserNotificationCenter.current().getNotificationSettings { settings in
-      logger.info("Notification settings: \(settings)")
-    }
-  }
-
-  func userNotificationCenter(_: UNUserNotificationCenter, willPresent _: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
-    logger.info("forground notification")
-    completionHandler([.alert, .sound])
-  }
-
-  func userNotificationCenter(_: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-    if response.notification.request.identifier == Constants.notificationIdentifier {
-      logger.info("handling notifications")
-      let userInfo = response.notification.request.content.userInfo
-      let nc = NotificationCenter.default
-      DispatchQueue.main.async {
-        nc.post(
-          name: dcNotificationViewChat,
-          object: nil,
-          userInfo: userInfo
-        )
-      }
-    }
-
-    completionHandler()
-  }
+	static let appCoordinator = AppCoordinator()
```

Comment:
is this formatting change needed? makes it hard to see if anything actually changed here